packages/material-ui/src/Select/Select.js

```diff
@@ -101,6 +101,7 @@ Select.propTypes = {
   classes: PropTypes.object.isRequired,
   /**
    * If `true`, the selected item is displayed even if its value is empty.
+   * Use along with renderValue function property to render custom value in empty selected value case.
```

Comment:
```suggestion
   * If you want to show a meaningful display, the `renderValue` property must be used to check for the case of an empty selected value and provide a custom render value.
```

Taking inspiration from https://github.com/mui-org/material-ui/issues/14742#issuecomment-469459392. What do you think?

Reply:
That works for me.  I'll update.